### DISCUSSION TITLE
Space+F with respect to currently open file?

### DISCUSSION THREAD

**Discussion by mxpv** [helix-editor/helix#10483] [Ideas] [answered]
Just a random thought. 

I know that this sort of backward incompatible, but would this behavior be more useful?

I found myself jumping between (rust) module's source files fairly often (using `Space+f`), especially while refactoring. But I don't typically `:cd` into each directory/subdirectory, so I don't use `space+F`.

It'd convenient to navigate to a next file within same package (like from a `random.rs` file back to `mod.rs` within same directory of a package). In this case `Space+F` command can look at currently opened buffer/file and display other source files relative to current's file directory.

Is this viable?

**Comment by kirawi** [comment, marked as answer]
There's a command for this: `file_picker_in_current_buffer_directory`

**Reply by mxpv** [reply]
Oh, wow! That's cool, I wasn't aware about this one. Thanks!

I can just do
```toml
[keys.normal.space]
F = "file_picker_in_current_buffer_directory"
```
